db-service/lib/cqn2sql.js

```diff
@@ -884,27 +884,35 @@ class CQN2SQLRenderer {
   }
 
   /**
-   * Renders a value into the correct SQL syntax of a placeholder for a prepared statement
+   * Renders a value into the correct SQL syntax or a placeholder for a prepared statement
    * @param {import('./infer/cqn').val} param0
    * @returns {string} SQL
    */
   val({ val, param }) {
+    const inline = !this.values || param === false
```

Comment:
Why did you pull that up here, if we still only need that down there in line 907, only if we reach that, of course?